### ISSUE TITLE
RosAria does not build due to CMakeList Changes

### ISSUE THREAD

**Issue by ongun-kanat** [amor-ros-pkg/rosaria#40]
New chages added with commit eed89e2662. Prevents RosAria to be built with Adept's packages on Ubuntu 14.04 with ros-indigo. Adept's packages should provide CMake or pkg-config instructions to find them.
The error:
```
CMake Error at rosaria/CMakeLists.txt:42 (find_package):
  By not providing "FindAria.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Aria", but
  CMake did not find one.

  Could not find a package configuration file provided by "Aria" with any of
  the following names:

    AriaConfig.cmake
    aria-config.cmake

  Add the installation prefix of "Aria" to CMAKE_PREFIX_PATH or set
  "Aria_DIR" to a directory containing one of the above files.  If "Aria"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Comment by reed-adept** [comment]
Fixed in a5e88b9

**Comment by reed-adept** [comment]
The idea is that if ARIA happens to provide CMake info then find_package will work, but it falls back on manually checking either for AMR original package or repackaged sourcedep package built during rosdep.
